modules/zip/zip_packer.cpp

```diff
@@ -115,9 +133,16 @@ void ZIPPacker::_bind_methods() {
 	BIND_ENUM_CONSTANT(APPEND_CREATE);
 	BIND_ENUM_CONSTANT(APPEND_CREATEAFTER);
 	BIND_ENUM_CONSTANT(APPEND_ADDINZIP);
+
+	BIND_ENUM_CONSTANT(COMPRESSION_NONE);
+	BIND_ENUM_CONSTANT(COMPRESSION_DEFAULT);
+	BIND_ENUM_CONSTANT(COMPRESSION_BEST);
+	BIND_ENUM_CONSTANT(COMPRESSION_BEST_SPEED);
```

Comment:
List bindings in the same order as the enum, so that the order in the class reference XML is consistent:

```suggestion
	BIND_ENUM_CONSTANT(COMPRESSION_DEFAULT);
	BIND_ENUM_CONSTANT(COMPRESSION_NONE);
	BIND_ENUM_CONSTANT(COMPRESSION_BEST_SPEED);
	BIND_ENUM_CONSTANT(COMPRESSION_BEST);
```